examples/src/models/chat/integration_minimax.ts

```diff
@@ -0,0 +1,60 @@
+import { HumanMessage } from "langchain/schema";
+import { ChatMinimax } from "langchain/chat_models/minimax";
+
+// Use abab5.5
+const abab5_5 = new ChatMinimax({
+  modelName: "abab5.5-chat",
+  botSetting: [
```

Comment:
Does the bot support system messages as well?

Reply:
The model itself does not support system messages, but its ‘botSetting’ parameters are similar to system message functions.